scripts/sync-css-variables.js

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Is this file part of the scope of the PR? 
We avoid having different scopes in 1 PR as it will be hard to revert changes if needed.

Reply:
It's not, but was needed for me to reformat theming files (for switch they are huge and to control all of the variables is becoming very hard). Since it's an optional tool which is not called anywhere (only manual execution), I don't think it can affect future development activities